modules/scanning-images.adoc

```diff
@@ -61,7 +61,7 @@ Scanner identifies vulnerabilities in images that contain the following Linux di
 | Distribution | Version
 
 | link:https://www.alpinelinux.org/[Alpine Linux]
-| `alpine:v3.2`, `alpine:v3.3`, `alpine:v3.4`, `alpine:v3.5`, `alpine:v3.6`, `alpine:v3.7`, `alpine:v3.8`, `alpine:v3.9`, `alpine:v3.10`, `alpine:v3.11`, `alpine:v3.12`, `alpine:v3.13`, `alpine:v3.14`, `alpine:edge`
+| `alpine:v3.2`, `alpine:v3.3`, `alpine:v3.4`, `alpine:v3.5`, `alpine:v3.6`, `alpine:v3.7`, `alpine:v3.8`, `alpine:v3.9`, `alpine:v3.10`, `alpine:v3.11`, `alpine:v3.12`, `alpine:v3.13`, `alpine:v3.14`, `alpine:v3.16`, `alpine:edge`
```

Comment:
alpine:v3.15 should be here, too

Reply:
I just checked, and I don't see a ticket for it either, but I see it's in the CHANGELOG for the 69 release. I guess it was missed no worries :)